modules/moving-the-gitops-operator-pod-to-infrastructure-nodes.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assembly:
+//
+// * gitops_workloads_infranodes/running-gitops-workloads-on-infrastructure-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="moving-the-gitops-operator-pod-to-infrastructure-nodes_{context}"]
+= Moving the {gitops-shortname} Operator pod to infrastructure nodes
+
+You can move the {gitops-shortname} Operator pod to the infrastructure nodes.
+
+.Prerequisites
+* {gitops-title} is installed in your cluster.
+
+.Procedure
+
+. Label existing nodes as infrastructure by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node-name> node-role.kubernetes.io/infra=
+----
++
+.Example output
+[source,terminal]
+----
+node/<node-name> labeled
+----
+
+. Edit the {gitops-title} `Subscription` resource to specify where the Operator should run:
++
+[source,terminal]
+----
+$ oc -n openshift-gitops-operator edit subscription openshift-gitops-operator
+----
+
+. Add `nodeSelector` and `tolerations` to the `spec.config` field in the `Subscription` resource:
++
+.Example Subscription
```

Comment:
@ekristova Good job! Suggest adding callouts as they have done here: https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/installing-ossm.html#ossm-config-operator-infrastructure-node_installing-ossm

Reply:
Thank you! I added the callouts